src/battle/battle_command.c

```diff
@@ -3994,7 +3994,7 @@ BOOL BtlCmd_TryYawn(BattleSystem *bsys, BATTLECONTEXT *ctx) {
     if (ctx->battleMons[ctx->battlerIdTarget].moveEffectFlags & (192 << 5)) {
```

Comment:
any flags for this one?

Reply:
yeah it's `MOVE_EFFECT_FLAG_YAWN` lol I'm just a dummy